src/applications/combined-debt-portal/medical-copays/components/AccountSummary.jsx

```diff
@@ -49,11 +50,14 @@ const AccountSummary = ({
           {`New charges: ${currency(newCharges)}`}
         </li>
       </ul>
+      <h3 className="vads-u-margin-top--2">Accounnt number</h3>
```

Comment:
Slight misspelling here! But the rest looks good!

Reply:
Good catch! Just fixed it!